FHEM/14_SD_WS.pm

```diff
@@ -471,8 +473,7 @@ sub SD_WS_Parse($$)
 			return "";
 		} else {
 			Log3 $name, 4, "$name: SD_WS37 checksum ok $checksum = ".SD_WS_binaryToNumber($bitData,32,39);
-			$id = SD_WS_binaryToNumber($bitData,0,7);
-			$id = sprintf('%02X', $id);           # wandeln nach hex
+			$id = substr($rawData,0,2);
```

Comment:
Das hat nicht mit dem Atech  zu tun oder?

Reply:
Bei Protokoll 38 ist kein Attribut "model" erforderlich. Das benötigen wir nur bei Protokoll 33.
> Das hat nicht mit dem Atech zu tun oder?
Hat es nicht, war nur eine kleine Verbesserung, die mir aufgefallen ist. Warum erst aus 8 Bit eine Dezimalzahl machen und dann in Hex wandeln, wenn wir gleich aus der der DMSG den Hex bekommen können.